03.2_school_prepare_OD_file.R

```diff
@@ -8,14 +8,27 @@ geography <- "lsoa"
 set.seed(12345)
 
 #########################
-# LOAD SCHOOL FLOW DATASET
+# LOAD SCHOOL FLOW DATASETS
 #########################
-sf11 <- read_tsv(file.path(path_inputs, "01_raw/02_travel_data/school/NPD_originals_PRIVATE/Spring_Census_2011.txt"))
-sf11 <- sf11[,c(1, 3:4, 6:10)]
-names(sf11) <- sub(paste0("_SPR11"), "", names(sf11))
-names(sf11) <- tolower(names(sf11))
-sf11 <- dplyr::rename(sf11, lsoa01cd = `llsoa`, bicycle = `cycle`, foot = `walk`)
+# Open and rename the 2010 and 2011 datasets
+for (i in (10:11) ) {
+  sf <- read_tsv(file.path(path_inputs, "01_raw/02_travel_data/school/lsoa/NPD_originals_PRIVATE", paste0("Spring_Census_20",i,".txt")))
```

Comment:
`read_tsv` is slightly faster and in my experience provides slightly better outputs - again not a blocker though but FYI see more about this **tidyverse** equivalent to a base function here: http://readr.tidyverse.org/reference/read_delim.html

Reply:
good to know – obviously it would be a pain to have to rewrite the code (e.g. I did try read.csv  at one point but there were some things that seem to do less well than read_csv).  But if read_tsv  work faster but lets the rest of the code stay as it is then that seems great